CHANGELOG.md

```diff
@@ -1,17 +1,17 @@
 # avromatic changelog
 
 ## v2.0.0 (unreleased)
-- Remove Virtus dependency.
-- Raise `Avromatic::Model::CoercionError` when attribute values can't be coerced to the target type.
+- Remove [virtus](https://github.com/solnic/virtus) dependency resulting in a 3x performance improvement in model instantation and 1.4x - 2.0x performance improvement in Avro serialization and Avromatic code simplification.
+- Raise `Avromatic::Model::CoercionError` when attribute values can't be coerced to the target type in model constructors and attribute setters. Previously coercion errors weren't detected until Avro serialization or an explicit call to `valid?`.
 - Prevent model instances from being constructed with unknown attributes by default. This can be disabled
-  by setting `Avromatic.allow_unknown_attributes` to `false`.
+  by setting `Avromatic.allow_unknown_attributes` to `false`. Previously unknown attributes were ignored.
```

Comment:
Does setting this to `false` have the side-effect that union members may be obscured by an earlier union member that is satisfied by a subset of the latter union member's attributes?